src/net_processing.cpp

```diff
@@ -2328,13 +2328,13 @@ void PeerManagerImpl::RelayInvFiltered(CInv &inv, const uint256& relatedTxHash,
 
 void PeerManagerImpl::RelayTransaction(const uint256& txid)
 {
+    const CInv inv{m_cj_ctx->dstxman->GetDSTX(txid) ? MSG_DSTX : MSG_TX, txid};
```

Comment:
usually it's only 10-12 peers, so, maybe improvements it un-noticeable by benchmark

Reply:
Well, no, not necessarily, MNs are highly connected. As an example on testnet
```
ubuntu@masternode-12:~$ dash-cli getpeerinfo | jq .[].id | wc -l
100
```

Reply:
ah yes, I completely forgot about MNs 🫣